### ISSUE TITLE
GammatoneBands error for sample rate other than 44100 Hz 

### ISSUE THREAD

**Issue by chrispikesonos** [pyfar/pyfar#373]
## General

* pyfar version: 0.5.0
* Python version: 3.9.7
* Operating System: MacOS 11.6.8
* Did you install pyfar via pip: yes

## Description

I tried to create a Gammatone filterbank at sampling rate 48kHz.

## What I Did

```
gfb = pf.dsp.filter.GammatoneBands((100, 20e3), sampling_rate=fs)
```

This gave the following error, triggered by the call at gammatone.py:278

```
ValueError: The sampling rates of the signal and Gammatone filter bank do not match
```

Changing gammatone.py:278 to the following appears to fix the issue:

```
real, imag = self.process(pf.signals.impulse(delay_samples + 3, sampling_rate=self.sampling_rate))
```

**Comment by chrispikesonos** [comment]
I don't seem to have permission to add labels, as requested by the template text.

**Comment by f-brinkmann** [comment]
Hi Chris, stoked that you are using pyfar and thanks for finding and fixing the bug. I opened a pull for that #374. I hope we will make a patch release soon.

**Comment by chrispikesonos** [comment]
Fab. Thanks a lot Fabian. I have just been tinkering so far but it's looking really useful. Thank you for your great work!

**Comment by f-brinkmann** [comment]
Fixed in 0.5.1
